### PR TITLE
[US-562026] Deprecation of Pega Search on Dockerhub

### PR DESCRIPTION
**Pull request by kingakowalska1**
Updated the README section for Search with the new location of the platform/search image. Clarified that the Elasticsearch plug-in is a deprecated solution.

### CODE REVIEW

File: charts/pega/README.md [hunk 1]
```diff
@@ -827,26 +829,28 @@ pegasearch:
...
+### For Pega Platform 8.5 and earlier:
+
+Use the following configuration to provision the legacy client-server Elasticsearch plug-in with a Pega-provided Docker image. This is a deprecated solution; as a best practice, update your deployment to Pega Platform version 8.6 or later and use SRS instead.
 
 Parameter   | Description   | Default value
 ---         | ---           | ---
-`image`   | Set the `pegasearch.image` location to a registry that can access the Pega search Docker image. The image is [available on DockerHub](https://hub.docker.com/r/pegasystems/search), and you may choose to mirror it in a private Docker repository. | `pegasystems/search:latest`
+`image`   | Set the `pegasearch.image` parameter to a registry that can access the Pega-provided `platform/search` Docker image. Download the image from the Pega repository, tag it, and push it to your local registry. For more information, see [Pega-provided Docker images](https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html). | `platform/search:latest`
```
Comment (taz-pega-work) [inline charts/pega/README.md:838]: maybe clarify not supported by adding a sentence so there's not question> Pega stopped maintaining this image in July 2023 due to its use of Elasticsearch version 5.6.x.\
Set the `pegasearch.image` parameter to...

**Review by taz-pega-work — APPROVED** [review]

File: charts/pega/README.md (hunk 1 above)
Comment (kotamsetti) [reply charts/pega/README.md:838]: @kingakowalska1 , are we sure about the path `/platform/search:latest` . please do check if the `latest` tag is available or not !!!

i get below error while pulling image by running `docker pull pega-docker.downloads.pega.com/platform/search:latest`

Error : Error response from daemon: manifest for pega-docker.downloads.pega.com/platform/search:latest not found: manifest unknown: The named manifest is not known to the registry.

Comment (kingakowalska1) [reply charts/pega/README.md]: @kotamsetti I updated the path to the image and added some relevant information on how to choose the images. Does this address your concerns? :) 

Thanks!
Kinga

**Review by kotamsetti — APPROVED** [review]
LGTM

**Review by misterdorito — APPROVED** [review]

**Pull request merged**